app/views/submissions/view.html.erb

```diff
@@ -79,7 +79,8 @@
             </option>
           <% end %>
         </select>
-        <strong>/<%= @userVersions.length %>
+        <strong>/<%= @userVersions.first.version %>
```

Comment:
May I understand the change here?

Reply:
The issue with using `@userVersions.length` is that when we delete a submission, the version numbers of the other submissions do not change.

So if we had 5 submissions initially and deleted the first version, the dropdown would now show `version: {2, 3, 4, 5} / 4`.

By using the version of the first (i.e. latest) submission, the dropdown will now show `{2, 3, 4, 5} / 5`.

An alternative would be to update the version numbers of other submissions instead, but it seems overkill.

Reply:
That's a sensible approach.